README.md

```diff
@@ -1,28 +1,35 @@
 # WikidataIdsDecorator
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/wikidata_ids_decorator`. To experiment with that code, run `bin/console` for an interactive prompt.
-
-TODO: Delete this and the text above, and describe your gem
+[Scraped](https://github.com/everypolitician/scraped) decorator which adds a `data-` attribute to Wikipedia links containing the Wikidata ID for the linked item, if relevant.
```

Comment:
This sentence is a little hard to grok, but I don't think it's accurate. AFAICS it actually adds a `wikidata` attribute, not a `data-` one. The "if relevant" is also a little hard to follow here. I think it's worth being a lot more explicit that the code actually goes and looks these up. Maybe something like?:

> When scraping a page from Wikipedia, this will attempt to look up the associated Wikidata item for each other Wikipedia page linked from within the page content, and adds a `wikidata="Qxxx"` attribute to each successfully resolved link.